pom.xml

```diff
@@ -5,17 +5,17 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.0.1</version>
+		<version>2.7.7</version>
```

Comment:
This repository should be kept for boot 3.x.   If you agree I can fix on merge.

Reply:
:picard-face-palm: